src/coreclr/interpreter/intops.h

```diff
@@ -21,6 +21,7 @@ typedef enum
     InterpOpBranch,
     InterpOpSwitch,
     InterpOpMethodToken,
+    InterpOpClassHandle,
```

Comment:
Would be nice to have same unified suffix for these. Either `Token` or `Handle`

Reply:
I propose Handle since the methodtoken one isn't actually a token.

Reply:
I fixed this and also renamed METHOD_DESC_TAG to METHOD_HANDLE_TAG for consistency